test/script/ref_script.cmake

```diff
@@ -0,0 +1,104 @@
+# Usage:
```

Comment:
What is this file for? Is it left over from testing/debugging?

Reply:
That's for ppl who wants to do ctest locally from a script, which can be used as reference for connivence.